src/randomization.ts

```diff
@@ -43,9 +43,9 @@ function swapItems<T>(stimuli: Array<T>, i1: number, i2: number) {
 }
 
 // A constraint a key (string) that is allowed a number of times in a row
-type ConstraintMapping = { [key:string]: number};
+type ConstraintMapping = Record<string, number>;
 
-type StringToUnknownMapping = {[key:string] : unknown};
```

Comment:
Changing these to use the `Record` type follows convention (and subjectively improves readability), but is otherwise functionally the same